### PR TITLE
fix the compatibility issue between PY2 and PY3

### PR DESCRIPTION
**Pull request by sandyhouse**
1. fix the compatibility issue between PY2 and PY3
2. some bug fixes
3. reformat the code

### CODE REVIEW

File: plsc/entry.py [hunk 1]
```diff
@@ -97,7 +98,9 @@ def __init__(self):
 
         self.fs_name = None
         self.fs_ugi = None
-        self.fs_dir = None
+        self.fs_remote_dir_for_save = None
+        self.fs_remote_dir_for_load = None
+        self.fs_local_dir_for_load = None
```
Comment (gavin1332) [inline plsc/entry.py]: check necessary

File: plsc/entry.py [hunk 2]
```diff
@@ -120,6 +123,8 @@ def __init__(self):
         self.model_save_dir = self.config.model_save_dir
         self.warmup_epochs = self.config.warmup_epochs
         self.calc_train_acc = False
+        self.fleet = None
```
Comment (gavin1332) [inline plsc/entry.py]: not necessary

File: plsc/entry.py [hunk 3]
```diff
@@ -97,7 +98,9 @@ def __init__(self):
 
         self.fs_name = None
         self.fs_ugi = None
-        self.fs_dir = None
+        self.fs_remote_dir_for_save = None
+        self.fs_remote_dir_for_load = None
```
Comment (gavin1332) [inline plsc/entry.py]: fs_checkpoint_dir

File: plsc/entry.py [hunk 4]
```diff
@@ -339,7 +367,9 @@ def _get_optimizer(self):
 
     def build_program(self,
                       is_train=True,
-                      use_parallel_test=False):
+                      use_parallel_test=False,
+                      param_attr=None,
```
Comment (gavin1332) [inline plsc/entry.py]: set these attrs by other functions

File: plsc/entry.py [hunk 5]
```diff
@@ -594,119 +673,155 @@ def predict(self):
...
-            emb = exe.run(main_program, feed=feeder.feed(data),
-                fetch_list=fetch_list, use_program_cache=True)
+            emb = exe.run(main_program,
+                          feed=feeder.feed(data),
+                          fetch_list=fetch_list,
+                          use_program_cache=True)
             print("emb: ", emb)
 
-    def test(self, pass_id=0):
+    def test(self, pass_id=0, run_with_train=False, initialized=False):
```
Comment (gavin1332) [inline plsc/entry.py]: use member variables

File: plsc/models/resnet.py [hunk 6]
```diff
@@ -92,51 +93,79 @@ def conv_bn_layer(self,
             stride=stride,
             padding=pad,
             groups=groups,
-            act=None,
             param_attr=fluid.param_attr.ParamAttr(
```
Comment (gavin1332) [inline plsc/models/resnet.py]: why remove this variable

Comment (sandyhouse) [reply plsc/models/resnet.py]: The default value of act for conv2d is none, so i remove it.

File: plsc/entry.py (hunk 3 above)
Comment (sandyhouse) [reply plsc/entry.py]: done

File: plsc/entry.py (hunk 4 above)
Comment (sandyhouse) [reply plsc/entry.py]: done

File: plsc/entry.py (hunk 5 above)
Comment (sandyhouse) [reply plsc/entry.py]: done

File: plsc/entry.py (hunk 1 above)
Comment (sandyhouse) [reply plsc/entry.py]: done

**Pull request merged**